src/aiod/config/settings.py

```diff
@@ -3,3 +3,7 @@
 
 api_base_url: str = API_BASE_URL
 latest_version: str = LATEST_VERSION
+
+server_url = "http://localhost/aiod-auth/"
```

Comment:
I think the name should be more meaningful. If I encounter `server_url` elsewhere, I would assume it refers to the REST API endpoint and not the authentication endpoint.
```suggestion
auth_server_url = "http://localhost/aiod-auth/"
```
use an IDE to correctly refactor across files